packages/gui/src/lib/classes.ts

```diff
@@ -0,0 +1,34 @@
+// Since CSS classes are denoted with "." this
+// breaks our lodash toPath-based getters/setters.
+// So, in our internal data structure we swap out
+// the "." with a "&" and then swap it back when
+// generating the code.
+const SPECIAL_INTERNAL_CSS_CLASS_SYNTAX = '&'
```

Comment:
This seems a bit hacky, but couldn't figure out a more elegant work around. AFAIK the ampersand isn't part of the CSS selector grammar which is why it's often used in CSS in JS libs, so it's following along with that 🤷